docs/modules/ROOT/pages/caching.adoc

```diff
@@ -15,3 +15,5 @@ endif::[]
 In previous versions of `jbang`, Java 10+ direct launch of `.java` was used, but since v0.6 `jbang` works with Java 8 and thus it needs to do a separate compile step. Besides now working with Java 8 this also allows to cache the compiled script and thus launch faster on consecutive runs.
 
 The caching goes to `~/.jbang/cache` by default, you can run `jbang cache clear` to remove all cache data from this folder.
+
+The default cache location can be overwritten by the environment variable `JBANG_CACHE_DIR`. If `$JBANG_DIR` environment variable is set, the `cache` folder will be placed there.
```

Comment:
I'd remove the `$` from `$JBANG_DIR` (on Windows it would be `%JBANG_DIR%`, so let's just use the name)

Reply:
Was an error, fixed.